Closet-Org/app/src/main/java/Organize/Closet/ArticleOfClothing.java

```diff
@@ -4,6 +4,10 @@ public interface ArticleOfClothing {
     String getStyle(); 
     String getName();
     String getOccasion();
-    String seasonalPurpose();
+    String getSeason();
```

Comment:
Be prepared. A lot of our files have `seasonalPurpose();` included, so this change could crash a lot of files in the main branch already.

Reply:
No worries I will make any changes necessary after I am done.